udf-compiler/src/main/scala/com/nvidia/spark/udf/CatalystExpressionBuilder.scala

```diff
@@ -255,6 +255,7 @@ object CatalystExpressionBuilder extends Logging {
         case And(Literal.TrueLiteral, c) => simplifyExpr(c)
         case And(c, Literal.TrueLiteral) => simplifyExpr(c)
         case And(Literal.FalseLiteral, _) => Literal.FalseLiteral
+        case And(_, Literal.FalseLiteral) => Literal.FalseLiteral
```

Comment:
Can be collapsed to a single line
```suggestion
        case and: And if and.containsChild(Literal.FalseLiteral) => Literal.FalseLiteral
```

Reply:
done